cluster/src/main/scala/kanaloa/reactive/dispatcher/ClusterAwareBackend.scala

```diff
@@ -0,0 +1,90 @@
+package kanaloa.reactive.dispatcher
+
+import java.util.UUID
+
+import akka.actor._
+import akka.cluster.routing.{ClusterRouterGroup, ClusterRouterGroupSettings}
+import akka.routing._
+import akka.pattern.ask
+import akka.util.Timeout
+import kanaloa.reactive.dispatcher.ClusterAwareBackend.RouteeRef
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
```

Comment:
If I understand, this type of Backend is meant to serve essentially as a Router to a series of remote Actors?